sympy/physics/mechanics/tests/test_joint.py

```diff
@@ -0,0 +1,113 @@
+from sympy.core.symbol import symbols
```

Comment:
I'd to see a test for a a PinJoint where the rotation vector is an arbitrary angle relative to each body's coordinate system, respectively. This should show that any angle can work.

Reply:
I didn't get it completely, can you please give an example of what you mean, it would ne really helpful.

Reply:
The pin joint axes should be able to be an axis that is oriented by two angles relative to the body's coordinate system, i.e. the rotation axis is not xhat, yhat, zhat of the body, nor does it lie in the xy, yz, xz planes. The pin joint axes should be arbitrarily oriented relative to body bodies it connects.

Reply:
Okay I'll add that test.